lib/collection/collectionDB.interface.ts

```diff
@@ -0,0 +1,34 @@
+export type ICollectionDBFieldsetLimits = {
+  token_limit: number,
+  limits_account_ownership?: number,
+  limits_sponsore_data_size?: number,
+  limits_sponsore_data_rate?: number,
+  owner_can_transfer?: boolean,
+  owner_can_destroy?: boolean
+};
+
+export type ICollectionDBFieldsetSchema = {
+  offchain_schema?: string,
+  const_chain_schema?: Object,
+  variable_on_chain_schema?: { collectionCover?: string },
+  schema_version?: string
+};
+
+export interface ICollectionDB extends ICollectionDBFieldsetLimits, ICollectionDBFieldsetSchema {
```

Comment:
Maybe ICollectionDBEntity or something else?

Reply:
`ICollectionDbEntity` is a beautiful name!)